common/client_test.go

```diff
@@ -314,7 +314,7 @@ func TestGetJWTProperty_Authenticate_Fail(t *testing.T) {
 	_, err := client.GetAzureJwtProperty("tid")
 	require.Error(t, err)
 	assert.True(t, strings.HasPrefix(err.Error(),
-		"default auth: azure-cli: cannot get access token: This is just a failing script"))
+		"default auth: azure-cli: cannot get account info"))
```

Comment:
Note: Earlier we used to do: 

`````
	if ee, ok := err.(*exec.ExitError); ok {
		return nil, fmt.Errorf("cannot get access token: %s", string(ee.Stderr))
	}
`````
and so we get the error from stderr which is "This is just a failing script" that we defined in test data.

We would need similar to for when we run: https://github.com/databricks/databricks-sdk-go/pull/1021/files#diff-eda1575b7fd7a235cdaeb9aefc3b7f84f8c4f5596726674ba187b39fab75ce13R166 but this would require another go sdk bump + release (or we can add the fix in go sdk and use pseudo version for bump)

I think, it should be okay to go with this change since we only need to check for the failure prefix and we can track to do this in next Go SDK and TF release.

Reply:
Test data ref for writing to stderr

```
if [ "yes" == "$FAIL" ]; then 
    >&2 /bin/echo "This is just a failing script."
    exit 1
fi
```

Reply:
+ @mgyucht for visibility on this as it was done in PR: https://github.com/databricks/databricks-sdk-go/pull/1021.